src/templates/validatingwebhookconfiguration.yaml.j2

```diff
@@ -0,0 +1,109 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  labels:
+    control-plane: kubeflow-training-operator
```

Comment:
I see upstream doesn't have that. Do we need it? Should we add a comment that this is diverging from upstream so if we need, we don't remove it in the future?

Reply:
This label is added when we render the manifests with `kustomize build apps/training-operator/upstream/overlays/kubeflow`.